R/stubs.R

```diff
@@ -24,5 +24,73 @@ sourceMarkers <- function(name, markers, basePath = NULL,
 
 #' @export
 navigateToFile <- function(file, line = 1L, column = 1L) {
-  callFun("navigateToFile", file, as.integer(line), as.integer(column))  
+  callFun("navigateToFile", file, as.integer(line), as.integer(column))
+}
+
+#' Replace Text within a Document
```

Comment:
Yeah, just matching package style here (pre-existing documentation used that format)